packages/create-nx-workspace/src/utils/nx/show-nx-warning.ts

```diff
@@ -0,0 +1,27 @@
+import { execSync } from 'child_process';
+import { resolve } from 'path';
+import { output } from '../output';
+
+/*
+ * Because we don't want to depend on @nrwl/workspace (to speed up the workspace creation)
+ * we duplicate the helper functions from @nrwl/workspace in this file.
+ */
```

Comment:
```suggestion
```

This comment can be removed, since this function isn't declared in @nrwl/workspace anymore.

Reply:
done